packages/app/src/app/pages/Dashboard/Content/routes/Settings/TeamSettings/RegistryPreset/Custom.tsx

```diff
@@ -71,15 +73,23 @@ export const CustomRegistryPreset = ({
         </Text>
       </div>
 
-      <CustomFormField label="Auth Token">
-        <Input
-          value={authKey}
-          required
-          onChange={e => setAuthKey(e.target.value)}
+      {authenticationType === AuthType.Basic ? (
+        <UsernamePassword
+          authKey={authKey}
           disabled={disabled}
-          type="password"
+          setAuthKey={setAuthKey}
         />
-      </CustomFormField>
+      ) : (
+        <CustomFormField label="Auth Token">
```

Comment:
It says only "Auth Token" here, not Token or Password

Reply:
In this case that's okay, because for Bearer authentication most registries call it "Auth Token"

Reply:
Ah yes, I was looking at the wrong label